docs/Using Price Feeds/historical-price-data.html

```diff
@@ -29,19 +29,93 @@
 
 To learn more about how price feeds update, see the [Decentralized Data Model](../architecture-decentralized-model/#aggregation-parameters) page.
 
-# Historical Rounds
+## Historical Rounds
 
-Data Feeds are updated in rounds. Rounds are identified by their `roundId`, which increases with each new round. The increase might not be monotonic. Knowing the `roundId` of a previous round allows contracts to consume historical price data.
+As shown in the [decentralized model](/docs/architecture-decentralized-model/), the consumer contracts call the proxy contract, which abstracts the underlying aggregator contract. The main advantage is to enable upgrades of the aggregator without impacting the consumer contracts. That also means that historical data can can be stored in different aggregators.
+As show in the following sequence diagram, to get historical data, you need to call the `getRoundData` [function](/docs/price-feeds-api-reference/#getrounddata) and provide `roundId` as a parameter.
+![getRoundData Sequence Diagram](/images/data-feed/getRoundData-sequence.png)
 
-The `updatedAt` data feed property is the timestamp of an answered round and `answeredInRound` is the round it was updated in. You can check `answeredInRound` against the current `roundId`. If `answeredInRound` is less than `roundId`, the answer is being carried over. If `answeredInRound` is equal to `roundId`, then the answer is fresh.
+Note that roundIds have different meanings in proxy contracts and in aggregator contracts. Let's clarify this:
+
+### `roundId` in Aggregator (aggregatorRoundId)
+
+Oracles provide periodic data updates to the aggregators. Data feeds are updated in **rounds**. Rounds are identified by their `roundId`, which increases with each new round. This increase may not be monotonic. Knowing the `roundId` of a previous round allows contracts to consume historical price data.
+In the following, we are going to name this roundId as **aggregatorRoundId** so that we differentiate it from the proxy's roundId.
```

Comment:
```suggestion
Oracles provide periodic data updates to the aggregators. Data feeds are updated in **rounds**. Rounds are identified by their `roundId`, which increases with each new round. This increase may not be monotonic. Knowing the `roundId` of a previous round allows contracts to consume historical price data.

The examples in this document name the aggregator `roundId` as `aggregatorRoundId` to differentiate it from the proxy `roundId`.
```

Add space between paragragraphs.